### PR TITLE
Tweak and clean up setuptools setup.py script

### PR DESCRIPTION
**Pull request by gwax**
- Switch README from Markdown to restructureText because PyPI doesn't like Markdown
- Extend setuptools to do more

### CODE REVIEW

**Comment by multigl** [comment]
:shipit:

File: setup.py [hunk 1]
```diff
@@ -1,34 +1,51 @@
...
-    ],
-
+    platforms=['any'],
     keywords='sqlalchemy postgresql orm temporal',
-    packages=find_packages(exclude=['docs', 'tests']),
-
-    install_requires=['sqlalchemy', 'psycopg2'],
+    classifiers=CLASSIFIERS,
+    install_requires=DEPENDENCIES,
+    tests_require=['pytest'],
```
Comment (jiffyclub) [inline setup.py]: Should this be `TEST_DEPENDENCIES`?

Comment (multigl) [inline setup.py]: it may not even be required until we figure out a reliable way for `python setup.py test` to work

Comment (jiffyclub) [inline setup.py]: They have docs about that: http://doc.pytest.org/en/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner

I've done with thing under the "Manual Integration" section and it worked. Haven't tried the pytest-runner plugin, that's new.

Comment (jiffyclub) [inline setup.py]: That said, I've only ever bothered to configure `python setup.py test` for projects with C extensions. For plain-Python projects I've stuck with `py.test`.

Comment (gwax) [inline setup.py]: Easy enough to do. Done.

Comment (multigl) [inline setup.py]: LGTM

**Pull request merged**